ouroboros-network-framework/src/Ouroboros/Network/Server2.hs

```diff
@@ -111,6 +111,7 @@ run :: forall muxMode socket peerAddr versionNumber m a b.
        , MonadCatch    m
        , MonadEvaluate m
        , MonadLabelledSTM  m
+       , MonadMask     m
```

Comment:
I think I was involved in writing this [commit](https://github.com/input-output-hk/ouroboros-network/pull/3281/commits/dc87fe9844e9d8f80aa6abcea7788616543cc8ec) too, isn't it; if that's the case one can add `Authors: ...` in the commit message (git does not support multi-authored commits.

Reply:
Yes you were! I will add that!

Reply:
Thanks, then a log reader might know where to ask.